lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -63,7 +63,11 @@ def verify(credit_card, options = {})
       def store(payment, options = {})
         post = {}
         add_payment(post, payment, options)
-        add_customer_object(post, payment, options)
+        phone = options.dig(:billing_address, :phone) .gsub(/\D/, '') unless options[:billing_address].nil?
```

Comment:
Can this live in a method for adding top level contact info and called where appropriate, including `add_customer_object` ? It looks like the only difference is where the `name` is passed, but it looks like everything else is the same.

Reply:
I'll have to think about where to put this. The issue is that in the `store` call it's top level on the `post` hash but in other calls it's in `post[:customer]` subhash

Reply:
Attempted a refactor to make it more clean and split it out into another commit. Let me know what you think!